libflux/flux-core/src/semantic/tests/vectorize.rs

```diff
@@ -421,3 +422,48 @@ fn vectorize_with_float_calls() -> anyhow::Result<()> {
     )?);
     Ok(())
 }
+
+#[test]
+fn vectorize_with_unary_add_sub() -> anyhow::Result<()> {
+    let pkg = vectorize(r#"(r) => ({ r with add: +r._value, sub: -r._value })"#).unwrap();
+
+    let function = get_vectorized_function(&pkg);
+
+    expect_test::expect![[r##"
+        (r) => {
+            return {r:{#C with _value: v[#B]} with add: +r:{#C with _value: v[#B]}._value:v[#B]:v[#B], sub: -r:{#C with _value: v[#B]}._value:v[#B]:v[#B]}:{#C with add: v[#B], sub: v[#B], _value: v[#B]}
+        }:(r: {#C with _value: v[#B]}) => {#C with add: v[#B], sub: v[#B], _value: v[#B]}"##]].assert_eq(&crate::semantic::formatter::format_node(
```

Comment:
I'm curious how you like this way of verifying vectorized code, or if you think there's a better way.

Reply:
I have a very hard time reading the string here, so I'm glad it's generated for me. I doubt I could write this by hand correctly without much trial and error.

For me, the exercise mostly comes down to making sure I see a `v[]` around either a scalar type or a type var in the right field positions. Other than that, the test gives me pass/fail on whether or not the function expression _can be vectorized_.

Seems sufficient.